### PR TITLE
Echange: fix display user addresses in dropdown select 

### PR DESCRIPTION
**Pull request by VitaliyShulik**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/MultiCurrencyWallet/blob/master/docs/CONTRIBUTING.md) guide
- [x] Good naming (as clear and simple as possible)
- [ ] Correct behavior if external API endpoints are down, return 404, 504 (or no answer), 401 errors (ddos simulation)
- [ ] I tested desktop/mobile resolution
- [ ] I tested light/dark theme
- [ ] I tested different languages
- [ ] I checked the functionality once again (**AFFECT MONEY**)
- [ ] I checked work on the Testnet
- [ ] I checked work on the Mainnet
- [x] I checked the **PR** once again

### Original issue

#4527

### CODE REVIEW

File: src/front/shared/pages/Exchange/AddressSelect/AddressSelect.tsx [hunk 1]
```diff
@@ -228,10 +207,11 @@ class AddressSelect extends Component<AddressSelectProps, AddressSelectState> {
       intl: { locale },
     } = this.props
     const ticker = this.getTicker()
+    const { coin } = getCoinInfo(ticker)
 
     feedback.exchangeForm.redirectedCreateWallet(ticker)
 
-    const url = localisedUrl(locale, `${links.createWallet}/${ticker}`)
+    const url = localisedUrl(locale, `${links.createWallet}/${coin}`)
```
Comment (NotEternal) [inline src/front/shared/pages/Exchange/AddressSelect/AddressSelect.tsx:214]: а если это токен, то выходит там будет **createWallet/WBTC** ? Но у нас уже используется **{ETH}WBTC** в адресах

Comment (VitaliyShulik) [reply src/front/shared/pages/Exchange/AddressSelect/AddressSelect.tsx:214]: #4558 да, создал отдельный issue

File: src/front/shared/redux/actions/core.ts [hunk 2]
```diff
@@ -368,8 +368,7 @@ const markCoinAsHidden = (coin, doBackup = false) => {
 const markCoinAsVisible = (coin, doBackup = false) => {
   const { hiddenCoinsList } = constants.localStorage
 
-  //@ts-ignore: strictNullChecks
-  const findedCoin = JSON.parse(localStorage.getItem(hiddenCoinsList)).find(
+  const findedCoin = JSON.parse(localStorage.getItem(hiddenCoinsList) || '""').find(
```
Comment (NotEternal) [inline src/front/shared/redux/actions/core.ts:371]: `JSON.parse(localStorage.getItem(hiddenCoinsList) || '[]').find(...`

со строкой `TypeError: "".find is not a function`

Comment (VitaliyShulik) [reply src/front/shared/redux/actions/core.ts:371]: окей, вроде проверял в консоле - поправлю

Comment (VitaliyShulik) [reply src/front/shared/redux/actions/core.ts]: готово

**Review by NotEternal — APPROVED** [review]

**Pull request merged**